checks/raw/testdata/.github/workflows/workflow-not-pinned.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
         # a pull request then we can checkout the head.
         fetch-depth: 2
 
```

Comment:
can we add tests for images hosted from other registries (gcr, ghr)? (unless they are not supported)